code/linalg/linalg.c

```diff
@@ -399,6 +400,31 @@ static mp_obj_t linalg_inv(mp_obj_t o_in) {
 
 MP_DEFINE_CONST_FUN_OBJ_1(linalg_inv_obj, linalg_inv);
 
+//| def norm(x: ulab.array) -> float:
+//|    """
+//|    :param ~ulab.array x: a vector or a matrix
+//|
+//|    Computes the 2-norm of a vector or a matrix, i.e., sqrt(sum(x*x)), however, without the RAM overhead.."""
+//|    ...
+//|
+
+the 2-norm of a matrix or vector, i.e., sqrt(sum(x*x)), however, without the RAM overhead.
```

Comment:
You have to take this line out, otherwise, the compiler fails.

Reply:
Sorry, paste which should be removed...

Commited.